### DISCUSSION TITLE
still pending

### DISCUSSION THREAD

**Discussion by ckokt** [Chia-Network/chia-blockchain#11200] [Support]
![geo pending](https://user-images.githubusercontent.com/2990501/163705926-6a6b60c2-24da-4463-a306-ef63e144a46a.jpg)

why still pending? already 4 days

**Comment by loppefaaret** [comment]
it can be several different factors - one being 0 fee, another being its a larger transaction, that potentially can hold multiple coin ID's, requiring more space in the blockchain to complete the transaction... 
Is your wallet reporting it is synced ? and are you on the latest version ?
